bindings/java/src/integration/com/apple/foundationdb/TransactionIntegrationTest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * TransactionIntegrationTests.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2023 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.apple.foundationdb;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionException;
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+/**
+ * Miscellanenous tests for Java-bindings-specific implementation of transactions
+ */
+@ExtendWith(RequiresDatabase.class)
+public class TransactionIntegrationTest {
+    public static final int API_VERSION = 720;
+	private static final FDB fdb = FDB.selectAPIVersion(API_VERSION);
+
+    @Test
+    public void testOperationsAfterCommit() throws Exception {
+		try (Database db = fdb.open()) {
```

Comment:
Something weird happened to the whitespace in this file, maybe a mix of tabs and spaces?

Reply:
Yes, it seems like a mix of tabs and spaces. I have to configure my Java formatter :-)